CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.1.0] - 2021-06-18
+
+### Added
+- Generic trustpilot widget block without badge.
+- Aditional props to widget
+
```

Comment:
Our release bot will add the date and version numbers when this PR is merged, no need to do it manually :)

```suggestion
## [Unreleased]
### Added
- Generic trustpilot widget block without badge.
- Aditional props to widget

```

Reply:
fixed